### ISSUE TITLE
Send thank you email notification to user submitting guest post

### ISSUE THREAD

**Issue by danielparas** [dative/guestentriesnotification#1]
hey @webrgp - thanks for this awesome plugin!

Just a quick question: I need to send a thank you email to the user creating the guest post. Am I correct in understanding that this is currently not possible, but is on the roadmap via the 'dynamic emails' feature?

Thanks
Dan

**Comment by webrgp** [comment]
Humm... do you mean a Craft user or from a email dynamically from a field?

Yes, currently it's not possible.

**Comment by danielparas** [comment]
Hey - thanks for your quick reply. I mean an email dynamically from a field posted via the form on the frontend.

**Comment by webrgp** [comment]
Is that email field in your channel?

**Comment by danielparas** [comment]
Yes it will be stored in the channel with the other fields posted from the form.

**Comment by danielparas** [comment]
Hey @webrgp - do you have an idea if/when this might be added? Or nothing is decided at this point?

**Comment by webrgp** [comment]
I am not sure. If you want to take a stab at it and send a pull request I would welcome it ;-)

**Comment by danielparas** [comment]
Hey @webrgp  - sorry for the late reply. Sure, right now I'm still waiting on a client to see if they want to proceed with the project. If they do I'll ping you here and take a shot at it. Thanks and happy new year! :)
